lib/getters/creditCardPaymentGetters.ts

```diff
@@ -0,0 +1,22 @@
+const getCardNumberMask = (tokenizedData): string => tokenizedData?.numberPart || ""
+const getId = (tokenizedData) => tokenizedData?.id || ""
+
+const getType = (creditCardData) => creditCardData?.cardType
+
+const getExpireMonth = (cardDetails): number => parseInt(cardDetails?.expiryDate.split("/")[0]) || 0
+
+const getExpireYear = (cardDetails): number => parseInt(cardDetails?.expiryDate.split("/")[1]) || 0
+
+const getNameOnCard = (cardDetails) => cardDetails?.nameOnCard
+
+const getAppliedTotal = (checkout) => checkout?.total
```

Comment:
there should already be a checkoutGetter.getTotal, use that instead

Reply:
checkoutGetter.getTotal is returning string value with $ symbol.
I have created getTotalToDisplay, which will return the string value to display on UI with $, and getTotal will return the numeric value to be used in request param.